AWSAuthSDK/Sources/AWSUserPoolsSignIn/UserPoolsUI/AWSUserPoolForgotPasswordViewController.m

```diff
@@ -158,13 +162,17 @@ - (void)setUp {
     [self.tableView reloadData];
     [AWSUserPoolsUIHelper setUpFormShadowForView:self.tableFormView];
     [self setUpBackground];
+    
+    // setup button background
+    [AWSUserPoolsUIHelper applyTintColorFromConfig:self.config
+                                            toView:self.updatePasswordButton];
 }
 
 - (void)setUpBackground {
     if ([AWSUserPoolsUIHelper isBackgroundColorFullScreen:self.config]) {
         self.view.backgroundColor = [AWSUserPoolsUIHelper getBackgroundColor:self.config];
     } else {
-        self.view.backgroundColor = [UIColor whiteColor];
+        self.view.backgroundColor = [AWSUserPoolsUIHelper getDefaultBackgroundColor];
```

Comment:
Without diving into the method, I'm not sure if this is a hardcoded default, or the one provided by the customer's configuration?

Reply:
I renamed things to `backgroundColor` and `secondaryBackgroundColor` to follow the current UIKit patterns. Intent should be more clear on the next revision.